addons/account_check_printing/models/res_partner.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    property_payment_method_id = fields.Many2one(
+        comodel_name='account.payment.method',
+        string='Payment Method',
+        company_dependent=True,
+        domain="[('payment_type', '=', 'outbound')]",
+        help="Preferred payment method when paying this vendor. This is used to filter vendor bills"
+             " by preferred payment method to register payments in mass. Use cases: create bank"
+             " files for batch wires, check runs.",
+    )
```

Comment:
We discussed it with AVW, and while I put it originally in `account`, it is useless with the payment methods.
It would only be useful if we could select the preferred payment journal too, meaning one field on `account.move` and one on `res.partner` for a feature not really well thought.